platform/mellanox/mlnx-platform-api/sonic_platform/chassis.py

```diff
@@ -33,6 +34,10 @@
 
 HWMGMT_SYSTEM_ROOT = '/var/run/hw-management/system/'
 
+MST_DEVICE_NAME_PATTERN = '/dev/mst/mt[0-9]*_pciconf0'
+MST_DEVICE_RE_PATTERN = '/dev/mst/mt([0-9]*)_pciconf0'
+CHIP_SPECTRUM1 = '53100'
```

Comment:
Would suggest to call it according to what this number stands for. For example SPECTRUM_1_PCI_ID
BTW why this is only supported for Spectrum-1 switched only? is it a limitation by hw-mgmt?
You mentioned later for now, so this is a known issue with redmine ticket that need to be solved?

Reply:
I don't think 53100 is the PCI id. Is it better to use SPECTRUM1_CHIP_ID? 
BTW, It is NOT supported on Spectrum1. I said "for now" because we raised a requirement to hw-mgmt so I think maybe in the future we'll have it.